docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+.. include:: ../../references.txt
+
+.. _pig-024:
+
+**************************
+PIG 24 - Authorship policy
+**************************
+
+* Authors: Bruno Khélifi, Thomas Vuillaume
+* Created: May 25th, 2022
+* Accepted:
+* Status: draft
+* Discussion:
+
+.. contents:: Table of Contents
+   :depth: 2
+
+Abstract
+========
+
+While the Gammapy library is more widely used by the community, a proper citation of the project including
+a policy about the authorship is necessary. This PIG addresses this issue by setting an authorship policy for the
+Gammapy project for each type of product.
+
+1. Introduction
+===============
+
+Gammapy started in 2013 and is now widely used in scientific publications. A proper citation scheme with correct authorship allows:
+
+- a proper citation of the used Gammapy release,
+- a proper recognition of the achieved work of any contributor,
+- compliance with the FAIR principles for Research Software (`FAIR4RS <https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs>`_).
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Reminder of the citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
```

Comment:
This is confusing. I would give Zenodo by default and mention other options such as HAL.

Reply:
Zenodo does not provide a long-term archive. One should not forget that! Pushing for Zenodo is then not necessary our task. This is a political message.
Giving the choice to people is more neutral